### PR TITLE
[#18][UI] As a user, I can response to textarea survey type

### LINKED ISSUE #18: [UI] As a user, I can response to textarea survey type
## Why
The application supports various types of survey items. After the user fills in all required survey items, the survey can be submitted and mark the survey as completed.

## Acceptance Criteria
- Display answer section for `textarea` survey type.
  image

- Store the user's response for survey submission

### PR DESCRIPTION
**Pull request by hanam1ni**
- Closes #18 

## What happened 👀

- Add textarea survey type
- Update browserslist version
 
## Proof Of Work 📹

https://user-images.githubusercontent.com/14077479/178208940-70b3cf19-51ca-4446-bbb5-e105bbb71fe5.mov

### CODE REVIEW

**Review by malparty — APPROVED** [review]
2 suggestions for improvement that are no blockers ;-) 
The rest LGTM! 🚀

File: src/components/SurveyAnswer/TextareaAnswer.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  setResponse,
+}: SurveyAnswerProps) => {
+  const handleAnswerChange = (
+    event: React.ChangeEvent<HTMLTextAreaElement>
+  ) => {
+    if (event.target.value === '') {
+      return setResponse(null);
+    }
+
+    const [answer] = question.answers;
```
[the remark this answers is not in the document]
Comment (hanam1ni) [reply src/components/SurveyAnswer/TextareaAnswer.tsx]: @malparty Updated the style for this and other places in [04a32e6](https://github.com/hanam1ni/react-surveyor/pull/86/commits/04a32e6456c7debedada45d40b74c1fba405fc1f)[04a32e6](https://github.com/hanam1ni/react-surveyor/pull/86/commits/04a32e6456c7debedada45d40b74c1fba405fc1f)

File: src/components/SurveyAnswer/index.tsx [hunk 2]
```diff
@@ -22,6 +23,8 @@ const SurveyAnswer = (props: SurveyAnswerProps) => {
       return <NpsAnswer {...props} />;
     case 'rating':
       return <RatingAnswer {...props} />;
+    case 'textarea':
+      return <TextareaAnswer {...props} />;
```
[the remark this answers is not in the document]
Comment (hanam1ni) [reply src/components/SurveyAnswer/index.tsx]: Updated in [04a32e6](https://github.com/hanam1ni/react-surveyor/pull/86/commits/04a32e6456c7debedada45d40b74c1fba405fc1f)[04a32e6](https://github.com/hanam1ni/react-surveyor/pull/86/commits/04a32e6456c7debedada45d40b74c1fba405fc1f)

[the remark this answers is not in the document]
Comment (malparty) [reply src/components/SurveyAnswer/index.tsx]: Nice! I forgot about Records in TypeScript!!! <3 

As often, you did it much better than what I had in mind! 😅 🚀

**Review by malparty — APPROVED** [review]

**Review by hoangmirs — APPROVED** [review]

**Pull request merged**